code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1008,10 +1008,11 @@ var/global/list/organ_damage_overlays = list(
 	if(reagents)
 
 		var/alien = 0 //Not the best way to handle it, but neater than checking this for every single reagent proc.
-		if(species && species.name == "Diona")
-			alien = 1
-		else if(species && species.name == "Vox")
-			alien = 2
+		if(src.species)
+			switch(src.species.type)
+				if(/datum/species/diona)	alien = 1
+				if(/datum/species/vox)	alien = 2
+				if(/datum/species/plasmaman)	alien = 3

```

Comment:
If someone edited the defines in the other file it would break this either way, unless I made the defines global